perf/perf_metric.py

```diff
@@ -82,12 +82,11 @@ def setUp(self):
 
     def _run_cmd(self, option):
         for line in self.list_of_metric_events:
-            cmd = "perf stat %s %s sleep 1" % (option, line)
-            rc, op = process.getstatusoutput(cmd, ignore_status=True,
-                                             shell=True, verbose=True)
+            cmd = "perf stat %s %s -C 0 sleep 1" % (option, line)
+            op = process.run(cmd, ignore_status=True, shell=True, verbose=True)
+            output = op.stdout.decode() + op.stderr.decode()
```

Comment:
cant I directly decode the whole output 
```
op = process.run(cmd, ignore_status=True, shell=True, verbose=True).decode()
```

Reply:
@abdhaleegit its not possible to directly decode the output
`AttributeError: 'CmdResult' object has no attribute 'decode'`

Reply:
ok it works only for output =  process.system_output().decode()   and not with process.run()

Reply:
correct and we need here process.run()

Reply:
we are good than